test/techs.e2e-spec.ts

```diff
@@ -108,8 +108,9 @@ describe("Techs Controller (e2e)", () => {
                     );
                 });
         });
-
-        it("POST - 201 adds new tech", async () => {
+    });
+    describe("POST voyages/:teamId/techs - add new tech item", () => {
```

Comment:
```suggestion
    describe("POST voyages/teams/:teamId/techs - add new tech item", () => {
```

Reply:
Thanks - I was just going to change these